css/index.css

```diff
@@ -12,6 +12,9 @@ body {
   --field-borders: #e0e0e0;
   --primary-button-bg: #1d1d1d;
   --primary-button-color: #eee;
+  --delete-button-background: #f3f3f3;
```

Comment:
```suggestion
```

Below I am setting background to `--elevation-background` so we won't need this variable